### PR TITLE
Adding HTTP client, Line and Line Groups handlers

### PR DESCRIPTION
**Pull request by EDDIE-X**
Adding HTTP client, Line and Line Groups handlers.

The following methods were implemented:

Line Group
- UpdateLineGroup

Line
- IsVirtualQActive

Adding gitignore file for .Net  projects.

README.md file was updated with build instructions.

No tests have been implemented yet.

### CODE REVIEW

File: VirtualQNet/Lines/LineAttributes.cs [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        public string Name { get; set; }
+
+        [JsonProperty("allow_callback")]
+        public bool AllowCallback { get; set; }
+
+        [JsonProperty("allow_sms")]
+        public bool AllowSms { get; set; }
+
+        [JsonProperty("service_waiters_count")]
+        public int ServiceWaitersCount { get; set; }
```
Comment (EDDIE-X) [inline VirtualQNet/Lines/LineAttributes.cs]: Shall we change this co ServiceCallersCount?

File: VirtualQNet/Lines/LineAttributes.cs [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+        [JsonProperty("virtualq_ewt")]
+        public int VirtualQEwt { get; set; }
+
+        [JsonProperty("virtualq_waiters_count")]
+        public int VirtualQWaitersCount { get; set; }
```
Comment (EDDIE-X) [inline VirtualQNet/Lines/LineAttributes.cs]: Shall we change this co VirtualQCallersCount?

**Comment by EDDIE-X** [comment]
Shall we change properties like:
public int Service**Waiters**Count { get; set; }
public int VirtualQ**Waiters**Count { get; set; }
To:
public int Service**Callers**Count { get; set; }
public int VirtualQ**Callers**Count { get; set; }
?

**Comment by nlib** [comment]
Please only change public interface methods from waiters to callers. Internally we'll keep waiters.

**Pull request merged**